clickhouse_mysql/writer/chwriter.py

```diff
@@ -86,13 +96,16 @@ def insert(self, event_or_events=None):
         schema = self.dst_schema if self.dst_schema else event_converted.schema
         table = None
         if self.dst_distribute:
-            table = TableProcessor.create_distributed_table_name(db=event_converted.schema, table=event_converted.table)
+            table = TableProcessor.create_distributed_table_name(
+                db=event_converted.schema, table=event_converted.table)
```

Comment:
maybe the naming comes from the library itself, but I find referring to the database as `schema` misleading.